compute_tools/src/spec.rs

```diff
@@ -744,7 +744,17 @@ pub fn handle_extension_neon(client: &mut Client) -> Result<()> {
     // - extension was just installed
     // - extension was already installed and is up to date
     let query = "ALTER EXTENSION neon UPDATE";
-    info!("update neon extension schema with query: {}", query);
+    info!("update neon extension version with query: {}", query);
+    client.simple_query(query)?;
+
+    Ok(())
+}
+
+#[instrument(skip_all)]
+pub fn handle_neon_extension_upgrade(client: &mut Client) -> Result<()> {
```

Comment:
Why not move whole `handle_extension_neon()`  into `post_apply_config`
instead of introducing new function?

Reply:
We do not need to install the neon extension and apply the schema changes. These are only applied on the first startup / apply config. As the newly-added background tasks will run every time the compute starts, I want to keep it as simple as possible.